### PR TITLE
15767-15763 fix approval type and relationship styling

### PR DESCRIPTION
**Pull request by jonathan-longe**
*Issue #:* bcgov/entity#15767 and bcgov/entity#15763

### ApprovalType
- Remove padding

### RelationshipPanel
- Remove padding
- Reduce font weight
- Modify font colour

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/approval-type/ApprovalType.vue [hunk 1]
```diff
@@ -32,7 +32,7 @@
           </v-form>
           <!-- REGISTRAR section -->
           <v-radio v-if="!isCourtOrderOnly"
-            id="registrar-radio" class="mb-0 pt-2"
+            id="registrar-radio" class="mb-n5 pt-2"
```
Comment (jonathan-longe) [inline src/components/approval-type/ApprovalType.vue:35]: Remove bottom padding

File: src/components/relationships-panel/RelationshipsPanel.vue [hunk 2]
```diff
@@ -6,41 +6,56 @@
...
             v-model="selectedRelationships"
             :value="RelationshipTypes.HEIR_LEGAL_REP"
-            :label="RelationshipTypes.HEIR_LEGAL_REP"
             :error="displayErrorState"
-          />
+          >
+            <template v-slot:label>
+              <span class="vuetify-label">{{ RelationshipTypes.HEIR_LEGAL_REP }}</span>
+            </template>
+          </v-checkbox>
```
Comment (jonathan-longe) [inline src/components/relationships-panel/RelationshipsPanel.vue:14]: Use the label slot to change theme of the label.

File: src/components/relationships-panel/RelationshipsPanel.vue [hunk 3]
```diff
@@ -119,4 +134,9 @@ export default class RelationshipsPanel extends Vue {
   align-items: stretch;
 }
 
+.vuetify-label {
+    color: $gray7;
+    font-weight: normal;
+}
```
Comment (jonathan-longe) [inline src/components/relationships-panel/RelationshipsPanel.vue:140]: Use CSS to set the style of the label as per Ethan's request

File: src/components/approval-type/ApprovalType.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/approval-type/ApprovalType.vue:35]: I think v-radio has a prop to remove extra bottom whitespace... Auto messages or something.

File: src/components/relationships-panel/RelationshipsPanel.vue (hunk 2 above)
Comment (severinbeauvais) [reply src/components/relationships-panel/RelationshipsPanel.vue:14]: Nice.

**Review by severinbeauvais — COMMENTED** [review]
I'll let others review and tomorrow I can publish it.

**Review by ketaki-deodhar — APPROVED** [review]
Looks good!

File: src/components/approval-type/ApprovalType.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/approval-type/ApprovalType.vue:35]: It's OK like this.

**Review by severinbeauvais — APPROVED** [review]

**Review by JazzarKarim — APPROVED** [review]
LGTM! 😃

File: src/components/relationships-panel/RelationshipsPanel.vue (hunk 2 above)
Comment (jonathan-longe) [reply src/components/relationships-panel/RelationshipsPanel.vue:14]: Thanks @severinbeauvais I did consider using `:deep()` but I thought this was simpler and easier to understand.  I'd be very happy to consider an alternative approach if using the slot makes you at all uncomfortable.

Comment (severinbeauvais) [reply src/components/relationships-panel/RelationshipsPanel.vue:14]: This is easier to understand. Leave it.

**Pull request merged**